citrination_client/models/design/tests/test_target.py

```diff
@@ -15,4 +15,5 @@ def test_target_initialization():
 
     # These initializations should not throw an error
     Target(name="Band gap", objective="Min")
-    Target(name="Band gap", objective="Max")
\ No newline at end of file
+    Target(name="Band gap", objective="Max")
```

Comment:
Looks like we changed the cases on Min and Max - is that an API change that would affect existing scripts users have written?

Reply:
No - the backend already converts to lower.

Reply:
Good catch, though!